rpcs3/Emu/Cell/SPUThread.h

```diff
@@ -570,7 +570,7 @@ class spu_thread : public cpu_thread
 
 	std::array<spu_int_ctrl_t, 3> int_ctrl; // SPU Class 0, 1, 2 Interrupt Management
 
-	std::array<std::pair<u32, std::weak_ptr<lv2_event_queue>>, 32> spuq; // Event Queue Keys for SPU Thread
+	std::array<std::pair<s64, std::weak_ptr<lv2_event_queue>>, 32> spuq; // Event Queue Keys for SPU Thread (-1 if not binded)
```

Comment:
It's possible to test weak_ptr for nullptr: https://stackoverflow.com/questions/45507041/how-to-check-if-weak-ptr-is-empty-non-assigned

Reply:
Thanks, changed it.